state/sqlite/sqlite_dbaccess.go

```diff
@@ -297,42 +297,41 @@ func (a *sqliteDBAccess) BulkGet(parentCtx context.Context, req []state.GetReque
 		return nil, err
 	}
 
-	var (
-		n    int
-		etag string
-	)
+	var n int
 	res := make([]state.BulkGetResponse, len(req))
 	for ; rows.Next(); n++ {
 		r := state.BulkGetResponse{}
-		r.Key, r.Data, etag, err = readRow(rows)
+		r.Key, r.Data, r.ETag, err = readRow(rows)
```

Comment:
eTag issues here too?

Reply:
This is another example of an etag error that's been fixed (and #2834 fixes more).

This was actually a much worse error because it caused BulkGet to return the wrong etag, as it was getting the address of a variable outside of the loop